src/scanoss/api/scanning/v2/scanoss_scanning_pb2_grpc.py

```
@@ -1,13 +1,35 @@
 # Generated by the gRPC Python protocol compiler plugin. DO NOT EDIT!
 """Client and server classes corresponding to protobuf-defined services."""
-
 import grpc
+import warnings
```

Ruff (0.8.2)

`warnings` imported but unused

Remove unused import: `warnings`

(F401)